lib/transmuxer/config.rb

```diff
@@ -5,11 +5,12 @@ class Config
     include Singleton
 
     attr_accessor :notifications_host
-    attr_reader :s3, :zencoder
+    attr_reader :s3, :zencoder, :cdn
```

Comment:
Another option could be simply adding an attr_accessor for `cdn_host`. 

With Zendcoder and S3 I can see having an actual object, because they represent concrete services and could be extracted out in time if we wanted to support additional file stores. However, a cdn host is sort of neutral in that it’s just a url, and a type of global setting that would sit in front of urls regardless of the file store. 

This could then be configured like so:

```ruby
Transmuxer.config do |c|
  c.cdn_host = "https://your-dist-url.cloudfront.net"
end
```

Any thoughts?